app/uk/gov/hmrc/merchandiseinbaggage/connectors/CurrencyConversionConnector.scala

```diff
@@ -27,13 +27,16 @@ import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class CurrencyConversionConnector @Inject() (
+  val appConfig: AppConfig,
   httpClient: HttpClient,
   @Named("currencyConversionBaseUrl") baseUrl: String
```

Comment:
Please remove this line as it is unnecessary. Also, please remove ConfigModule [here](https://github.com/hmrc/merchandise-in-baggage/blob/DDCE-5000/conf/application.conf#L30-L35) and delete the file [here](https://github.com/hmrc/merchandise-in-baggage/blob/DDCE-5000/app/uk/gov/hmrc/merchandiseinbaggage/config/ConfigModule.scala), thanks.

Reply:
ohh yeah I missed removing ConfigModule, thanks Taiwo! do I need to also remove the currency-conversion configs (protocol, hots, port) from application.conf or is it used for something else?

Reply:
No worries. No, don't remove those ones. `ServicesConfig` in bootstrap uses it to build the url.

Reply:
If you look into `servicesConfig.baseUrl` you will see this :)